exercises/J-array-get-set/exercises2.js

```diff
@@ -7,6 +7,7 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
+let end=numbers.push(4);
 
```

Comment:
here we don't assign it to a variable we just push it on our existing array by name numbers like this :
numbers.push(4);

Reply:
According to the exercise it should be
```
numbers[3] = 4
```
Although
```
numbers.push(4)
```
is honestly probably better.

You don't need to assign it to a variable in any case, though.